tools/yarn.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import os
+
+import b
+from dockerise import WrapPty, run_on_docker
```

Comment:
Since we dont actually run this on docker
```suggestion
from dockerise import WrapPty
```